### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by Josh012422**
The purpose of this PR is to add the project's README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Description
+
+A personal project to learn how to make a cli with cobra and go
```
Comment (cortizqgithub) [inline README.md:3]:
```suggestion
A personal project to learn how to make a **_cli_** with **_cobra_** and go
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+Make to do your terminal life easier
```
Comment (cortizqgithub) [inline README.md:5]: Put a point at the end.

```suggestion
Make your terminal life easier.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+# Installation
+
+To install, clone the repo the simply run:
```
Comment (cortizqgithub) [inline README.md:9]:
```suggestion
To install, clone the repo then simply run:
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+$ make
+```
+
+Let it finish, then:
+
+```bash
+$ echo 'export PATH="$HOME/go/bin:$PATH"' >> ~/.bash_profile
+```
+
+If you don't have `make` install search Google.
```
Comment (cortizqgithub) [inline README.md:21]:
```suggestion
If you don't have `**_make_**` installed in your machine, then search Google.
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+Hint: If your on windows installl choco, then run:
```
Comment (cortizqgithub) [inline README.md:23]:
```suggestion
**Hint:** If you're on '**_Windows_**' then install choco, then run:
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+```bash
+choco install make
+```
+Hint: If you are using termux, just run:
```
Comment (cortizqgithub) [inline README.md:28]:
```suggestion
**_Hint:_** If you are using termux, just run:
```

File: README.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+```bash
+$ pkg install make
+```
+
+# Usage
+
+The "main" command is gocharm, there are currently 2 commands (There will be more), a list:
```
Comment (cortizqgithub) [inline README.md:36]:
```suggestion
The '**_main_**' command is `gocharm`, there are currently two commands (There will be more), a list:
```

File: README.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+1. time
+2. tasks
+
+Each one with his own functionality.
+
+## Time Command
+
+This command displays the REAL TIME of any city you want, For example:
```
Comment (cortizqgithub) [inline README.md:45]:
```suggestion
This command displays the REAL TIME of any city you want, for example:
```

File: README.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+$ gocharm time -t "America/New_York" -1
+```
+
+The "-t" flag tells gocharm what city to look the time of, like the timezone.
+
+The "-1" flag tells gocharm to display the time in 12-Hour Format
+
+### Compare Subcommand
+
+There's a subcommand of time that lets you display the time of 2 different timezones, The Compare Command.
```
Comment (cortizqgithub) [inline README.md:57]:
```suggestion
There's a subcommand of time that lets you display the time of two different timezones. The Compare Command.
```

File: README.md [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+
+The tasks command is a small and simple task manager, it lets you add and save tasks to a file in your systems, so if you prompted title wrong you can go and modify it.
+
+```bash
+$ gocharm tasks add
+```
+
+The above command will add a new task with a number a save it in your system.
+
+Note: There's also other subcommands like: list, view, complete (Not finished yet) but for this README the above one is enough.
```
Comment (cortizqgithub) [inline README.md:75]:
```suggestion
NOTE: There's also other subcommands like '**_list_**', '**_view_**', '**_complete_**' .
```

File: README.md (hunk 1 above)
Comment (Josh012422) [reply README.md:3]: I like, I will add it.

File: README.md (hunk 3 above)
Comment (Josh012422) [reply README.md:9]: OK

File: README.md (hunk 4 above)
Comment (Josh012422) [reply README.md:21]: OK

File: README.md (hunk 2 above)
Comment (Josh012422) [reply README.md:5]: OK

File: README.md (hunk 6 above)
Comment (Josh012422) [reply README.md:28]: OK

File: README.md (hunk 7 above)
Comment (Josh012422) [reply README.md:36]: ok

File: README.md (hunk 8 above)
Comment (Josh012422) [reply README.md:45]: OK

File: README.md (hunk 9 above)
Comment (Josh012422) [reply README.md:57]: ok

**Pull request merged**